content/docs/pipelines/overview/concepts/pipeline.md

```diff
@@ -5,17 +5,16 @@ weight = 10
 +++
 
 A *pipeline* is a description of a machine learning (ML) workflow, including all
-of the components in the workflow and how the components relate to each other in
+of the [components](/docs/pipelines/overview/concepts/component/) in the workflow and how the components relate to each other in
 the form of a [graph](/docs/pipelines/concepts/graph/). The pipeline
 configuration includes the definition of the inputs (parameters) required to run
 the pipeline and the inputs and outputs of each component.
 
 When you run a pipeline, the system launches one or more Kubernetes Pods
-corresponding to the steps (components) in your workflow (pipeline). The Pods
+corresponding to the [steps (components)](/docs/pipelines/overview/concepts/step/) in your workflow (pipeline). The Pods
```

Comment:
Suggestion: Would you please move (components) outside of the link for steps?

```
[steps](/docs/pipelines/overview/concepts/step/) (components)
```

Reply:
Thanks @joeliedtke for reviewing this PR. I have modified the file as what you suggested. Please review/approve. Feel free to let me know if further change is needed.